mapie/classification.py

```diff
@@ -0,0 +1,305 @@
+from __future__ import annotations
+from typing import Optional, Union, Iterable
+
+import numpy as np
+from sklearn.utils import check_X_y, check_array
+from sklearn.utils.validation import check_is_fitted
+from sklearn.base import clone
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import Pipeline
+
+from ._typing import ArrayLike
+from .utils import check_null_weight, fit_estimator
+from .utils import check_n_features_in, check_alpha
+
+
+class MapieClassifier (BaseEstimator, ClassifierMixin):  # type: ignore
+    """
+    Prediction intervals for classification
+    using conformal predictions (in development).
+
+    Parameters
+    ----------
+    estimator : Optional[ClassifierMixin]
+        Any classifier with scikit-learn API
+        (i.e. with fit and predict methods), by default None.
+        If ``None``, estimator defaults to a ``LogisticRegression`` instance.
+
+    method: str, optional
+        Method to choose for prediction interval estimates.
+        Choose among:
+
+        - "naive", based on training set scores
+
+        By default "naive".
+
+    cv: Optional[str]
+        The cross-validation strategy for computing scores :
+
+        - ``None``, to use mapie for fitting.
```

Comment:
Décris un peu plus explicitement ce que ça va faire : "``None``, `MapieClassifier` will be used for fitting the base model" ou quelque chose comme ça.